169.py

```diff
@@ -1,4 +1,21 @@
+# class Solution:
+#     def majorityElement(self, nums: List[int]) -> int:
+#         sorted_nums = sorted(nums)
+#         return sorted_nums[len(nums) // 2]
+
+# O(n) time complexity.
 class Solution:
     def majorityElement(self, nums: List[int]) -> int:
-        sorted_nums = sorted(nums)
-        return sorted_nums[len(nums) // 2]
+        count = [nums[0], 1]
```

Comment:
`count[0]` and `count[1]` are not descriptive. I would rather declare two variables `current_majority` and `current_count` for better readability.